src/components/Expander/Expander.tsx

```diff
@@ -47,11 +62,11 @@ const StyledPanelExpansionTitle = styled(
   ${allStates('cursor: pointer;')}
 `;
 
-interface PanelExpansionState {
+interface ExpanderState {
   openState: boolean;
 }
 
-export interface PanelExpansionProps extends PanelProps {
+export interface ExpanderProps extends CommonExpanderProps {
```

Comment:
ExpanderBaseProps? BaseExpanderProps? SharedExpanderProps? Common is not bad either, just make the decision count since it will be difficult to change it afterwards.

Reply:
I made the change and it is now `SharedExpanderProps` that sounds better as `CommonExpanderProps` sounded... too common.